CHANGES.rst

```diff
@@ -3,11 +3,11 @@ CHANGES
 =======
 
 2.0 (unreleased)
-------------------
+----------------
 
 - Drop support for Python 2.6, 3.2, 3.3.
```

Comment:
Maybe somewhen, somebody forget to add 3.4 here?

Reply:
Probably 3.4 was never supported by this package.